crypto/kyber/p_kyber512_asn1.c

```diff
@@ -103,11 +103,11 @@ static int kyber512_pub_cmp(const EVP_PKEY *a, const EVP_PKEY *b) {
 }
 
 static int kyber512_size(const EVP_PKEY *pkey) {
```

Comment:
Are these functions exercised in a test function anywhere?

Reply:
yes, in `crypto/kyber/p_kyber_test.cc`